db/migrate/20150827092935_add_opt_in_field_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddOptInFieldToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :opt_in_for_research, :boolean, default: true
```

Comment:
@asmega yeah the default, i have been told, is to be set to true. The user will have to opt-out by unchecking the box. Yikes! haha